releases/v15_0.rst

```diff
@@ -0,0 +1,28 @@
+#######################
+LSST Stack 15.0 Release
+#######################
+
+Version 15.0 of the LSST Science Pipelines is now available. It is based on 2018 weekly 10.
+
+LSST Pipelines
+==============
+
+Full details of the LSST Science Pipelines 15.0 release can be found
+in the :ref:`extensive release notes <release-v15-0>`.  We have also
+published a `Characterization Metric Report <https://ls.st/DMTR-62>`_.
+
+Qserv and Data Access
+=====================
+
+The database team is on a separate release cadence.
+
+Science User Interface
+======================
+
+The Science User Interface (SUI) team releases their codebase independently.
```

Comment:
Similar comments here to the above.

Let's remove both of these blocks, and instead writes something like:

```
Other LSST Software
===================

In addition to the Science Pipelines, the Data Management team produces data archive and user interface tools in support of the LSST construction effort.
These are released separately.
Refer to https://www.lsst.org/about/dm for details.
```

Reply:
I really like this idea.